nucliadb/nucliadb/ingest/orm/brain.py

```diff
@@ -59,11 +59,10 @@
 FilePagePositions = Dict[int, Tuple[int, int]]
 
 
-PROCESSING_STATUS_PB_TYPE_TO_NAME_MAP = {
-    PBBrainResource.ERROR: ResourceProcessingStatus.ERROR.name,
-    PBBrainResource.EMPTY: ResourceProcessingStatus.EMPTY.name,
-    PBBrainResource.PROCESSED: ResourceProcessingStatus.PROCESSED.name,
-    PBBrainResource.PENDING: ResourceProcessingStatus.PENDING.name,
+METADATA_STATUS_PB_TYPE_TO_NAME_MAP = {
+    Metadata.Status.ERROR: ResourceProcessingStatus.ERROR.name,
+    Metadata.Status.PROCESSED: ResourceProcessingStatus.PROCESSED.name,
+    Metadata.Status.PENDING: ResourceProcessingStatus.PENDING.name,
```

Comment:
Empty is missing ?

Reply:
`EMPTY` is not part of the `Metadata.Status` [proto definition](https://github.com/nuclia/nucliadb/blob/main/nucliadb_protos/resources.proto#L263). I added `BLOCKED` and `EXPIRED` though, just in case